3_javascript/javascript1/week3/JS1-W3

```diff
@@ -0,0 +1,121 @@
+////////////////////////////Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+
+console.log(names);
+
+////////////////////////////When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function timeCalculator() {
+    let travelTimeInHours = travelInformation.destinationDistance / travelInformation.speed;
+    const hours = Math.floor(travelTimeInHours);
+    const minutes = Math.round((travelTimeInHours - hours) * 60);
+    
+    return {
+      hours,
+      minutes,
+    };
+  }
+  
+  const { hours, minutes } = timeCalculator(travelInformation);
+  
+  console.log(`${hours} hours and ${minutes} minutes`);
+  
```

Comment:
🌟 I admire how you've approached the time calculation. This ensures you can get both hours and minutes without any overlap!
😊 Consider passing `travelInformation` as a parameter to the `timeCalculator()` function. This would make your function even more versatile and usable with different travel information.
📝 A tiny adjustment: For `travelTimeInHours`, it's more fitting to use `const` instead of `let` since the value isn't being reassigned.